tutorial/12-completed.md

```diff
@@ -0,0 +1,23 @@
+---
+ms.localizationpriority: medium
+---
+
+<!-- markdownlint-disable MD041 -->
+
+You've completed the .NET Microsoft Graph tutorial. Now that you have a working app that calls Microsoft Graph, you can experiment and add new features. Visit the [Overview of Microsoft Graph](/graph/overview) to see all of the data you can access with Microsoft Graph.
+
+## .NET samples
```

Comment:
Should we also highlight "other" types of samples? Front-end, etc?

Reply:
My plan is to keep them by language. I only included .NET ones here, but we'll do a JavaScript one that's similar, and then link to all of our JS/TS samples, including FE stuff (and MGT).